keyboards/yellowbrickroad/readme.md

```diff
@@ -0,0 +1,32 @@
+# yellowbrickroad
+
+![yellowbrickroad](https://i.imgur.com/sy1wYZoh.jpg)
+
+"There’s no place like home.” I designed this keyboard since I wanted to go back to Kansas, LOL.
+
+yellowbrickroad is a MIDI keyboard which has 53 keys (= 4 octaves+ ) placed spirally. Raspberry Pi Pico is used as the "brain" of yellowbrickroad.
+
+Size: 20 cm (approx 8 inches) x 23 cm (approx 9 inches).
+It fits in an A4 / Letter size paper.
+
+* Keyboard Maintainer: [3araht](https://github.com/yourusername)
+* Hardware Supported: yellowbrickroad keyboard, a MIDI keyboard.
+* Hardware Availability: [Yushakobo](https://shop.yushakobo.jp/collections/keyboard/products/3442), [BOOTH](https://3araht.booth.pm/). Click [here](https://www.tenso.com/en/static/lp_shop_booth) for BOOTH overseas shipping!
+
+Make example for this keyboard (after setting up your build environment):
+
+    make yellowbrickroad:default
+
+Flashing example for this keyboard:
+
+    make yellowbrickroad:default:flash
+
+Run this while Raspberry Pi Pico is in bootloader mode.
+## Bootloader
```

Comment:
```suggestion

## Bootloader
```

Reply:
Not sure what this means. I added one empty row and committed but didn't help.